### ISSUE TITLE
ORF: multiple treatments

### ISSUE THREAD

**Issue by congcongruc** [microsoft/EconML#430]
Hi!

I would like to ask whether ORF supports multiple treatments as follows:
(1) Multiple discrete treatments 
(2) Multiple continuous treatments
(3) Multiple discrete and continuous treatment (e.g., T1 = gender, T2 = Income)

It seems that multiple discrete treatments are supported (as saw previously in a usage example in the OrthoForest Jupyter Notebook), but I'm not sure whether (2) and (3) are feasible, and how to implement them in code. Any ideas? Thanks a lot!!

BTW, it seems that the OrthoForest Jupyter Notebook page link (listed below) is not accessible recently. I don't know whether this is due to my Internet problem or something else.
https://github.com/Microsoft/EconML/blob/master/notebooks/Orthogonal%20Random%20Forest%20Examples.ipynb

**Comment by kbattocchi** [comment]
ORF supports multiple continuous treatments, or a single discrete treatment (with arbitrarily many classes).  If you want to run on multiple discrete treatments, you can combine them into a single logical treatment first by considering each unique combination of treatment values to be a different treatment.

**Comment by congcongruc** [comment]
Thanks!! Just two follow-up clarification questions.

1. I didn't find an ORF multiple continuous treatment example online. But following a similar example in DML, could I specify the treatment (T) to something like this: np.concatenate((T1.reshape(-1,1), (T2.reshape(-1,1)) ), axis=1)? Is this a correct way to specify multiple continuous treatments in ORF?

2. So it seems that ORF only supports multiple treatments where all treatments are continuous or discrete. The combination of continuous treatment and discrete treatment is not supported. Is this interpretation correct? Thanks!

**Comment by kbattocchi** [comment]
Yes, that's right on both counts. For multiple continuous treatments, T should be an array of shape (n, d_t), which concatenating the separate arrays does for you.  And we support only discrete or continuous treatments, but not both simultaneously.

**Comment by congcongruc** [comment]
Very useful explanations! Thanks!
